common/lib/xmodule/xmodule/modulestore/tests/test_split_modulestore.py

```diff
@@ -888,6 +889,25 @@ def test_block_generations(self, _from_json):
         version_history = modulestore().get_block_generations(second_problem.location)
         self.assertNotEqual(version_history.locator.version_guid, first_problem.location.version_guid)
 
+    def test_make_course_usage_key(self):
```

Comment:
Oh right. I was just thinking about CCX -- I forgot there were no explicit tests for this method.